agent/fw_drupal.c

```diff
@@ -838,11 +852,11 @@ void nr_drupal_enable(TSRMLS_D) {
     nr_php_wrap_user_function(NR_PSTR("module_invoke"),
                               nr_drupal_wrap_module_invoke TSRMLS_CC);
 #if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
-     && !defined OVERWRITE_ZEND_EXECUTE_DATA
-    nr_php_wrap_user_function_before_after_clean(NR_PSTR("module_invoke_all"),
-                                                 nr_drupal_wrap_module_invoke_all_before,
-                                                 nr_drupal_wrap_module_invoke_all_after,
-                                                 nr_drupal_wrap_module_invoke_all_clean);
+    && !defined OVERWRITE_ZEND_EXECUTE_DATA
+    nr_php_wrap_user_function_before_after_clean(
+        NR_PSTR("module_invoke_all"), nr_drupal_wrap_module_invoke_all_before,
+        nr_drupal_wrap_module_invoke_all_after,
+        nr_drupal_wrap_module_invoke_all_clean);
```

Comment:
This is just the formatting change, right?

Reply:
yes